vllm/v1/worker/tpu_model_runner.py

```diff
@@ -158,6 +159,23 @@ def __init__(
         # Range tensor with values [0 .. self.max_num_tokens - 1].
         # Used to initialize positions / context_lens / seq_lens
         self.arange_np = np.arange(self.max_num_tokens, dtype=np.int32)
+        self.paddings = self._get_paddings()
+
+    def _get_paddings(self) -> list[int]:
```

Comment:
It will be great if we can have unit tests for these functions which only exist in TPU platform, cc @yarongmu-google